mobile_app/lib/cafe_image.dart

```diff
@@ -22,20 +23,75 @@ class CafeImage extends StatelessWidget {
       child: CachedNetworkImage(
         imageUrl: '$path$group$uri$query',
         fit: BoxFit.cover,
-        progressIndicatorBuilder: (context, url, downloadProgress) {
-          return Center(
-              child: SizedBox(
-                  width: size * 0.1,
-                  height: size * 0.1,
-                  child: CircularProgressIndicator(
-                      color: Colors.black12,
-                      strokeWidth: size > 319 ? 5 : 2,
-                      value: downloadProgress.progress)));
-        },
+        progressIndicatorBuilder: (context, url, downloadProgress) => Center(
+            child: DelayedWidget(
+                key: ValueKey(image.id),
+                child: _CafeImageProgressIndicator(
+                    contentSize: size,
+                    progress: downloadProgress.progress ?? 0.0))),
         errorWidget: (context, error, stackTrace) {
           return Center(child: Text('no image'));
         },
       ),
     );
   }
 }
+
+class _CafeImageProgressIndicator extends StatelessWidget {
+  static const _color = Colors.black12;
+
+  final double contentSize;
+  final double progress;
+  _CafeImageProgressIndicator(
+      {required this.contentSize, required this.progress});
+
+  double get _displayedProgress => progress.clamp(0.1, 1.0);
+  double get _desiredSize => contentSize * 0.25;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+      Icon(Icons.image,
+          color: _color, size: _roundToMultipleOfFour(_desiredSize * 0.75)),
+      SizedBox(height: _roundToMultipleOfFour(_desiredSize * 0.05)),
```

Comment:
_getDelayedSize 함수로 분리하는 건 어떤지 의견 여쭤봅니다 ㅎㅎ
```suggestion
       Icon(Icons.image, color: _color, size: _getDelayedImageSize(_desiredSize).icon),
      SizedBox(height: _getDelayedImageSize(_desiredSize).progress),
```

```
DelayedImage _getDelayedImageSize(double size) {
    double iconSize = 0.75;
    double progressSize = 0.5;
    return DelayedImage(
        icon: _roundToMultipleOfFour(size * iconSize),
        progress: _roundToMultipleOfFour(size * progressSize));
 }


class DelayedImage {
  double icon;
  double progress;

  DelayedImage({required this.icon, required this.progress});
}
```